test-commons/src/main/java/com/sonarsource/checks/verifier/LineIssues.java

```diff
@@ -141,7 +141,8 @@ public String toString() {
     }
     Double effort = effortToFix();
     if (effort != null) {
-      out.append(" [[effortToFix=").append(effort.longValue()).append("]]");
+      String displayedEffort = effort.toString().replaceFirst(".0$", "");
```

Comment:
@alban-auzeill why `.replaceFirst(".0$", "")`?

Reply:
Error . should be replaced by \\.